ruma-api-macros/src/derive_outgoing.rs

```diff
@@ -114,6 +119,12 @@ pub fn expand_derive_outgoing(input: DeriveInput) -> syn::Result<TokenStream> {
     }
 }
 
+/// Keep any `serde(untagged)` or `non_exhaustive` attributes found and
```

Comment:
Not necessarily just `untagged`.

```suggestion
/// Keep any `serde` or `non_exhaustive` attributes found and
```